### ISSUE TITLE
Rubrik Verbundene Autoren umbenennen in: Verbundene Personen und Institutionen

### ISSUE THREAD

**Issue by coeh** [ubtue/tuefind#2639]
Beim Verfassen einer Erläuterung zu den Personenseiten für das neue IxTheo-Design ist uns aufgefallen, dass die Rubrik "Verbundene Autoren" etwas unpräzise benannt ist, da dort auch Körperschaften oder Konferenzen mit entsprechendem GND-Satz ausgewertet werden. Erfahrungsgemäß sorgt das auch bei Wissenschaftler*innen gelegentlich für Verwirrung, wenn sie ihre Personenseite anschauen.

Wir schlagen daher vor, die Rubrik in "Verbundene Personen und Institutionen" umzubenennen.

@thefass wird das Thema in die IT-Besprechung einbringen.

**Comment by relhei** [comment]
@coeh Ich stimme der vorgeschlagenen Lösung "Verbundene Personen" zu und einer entsprechenden Anpassung auch im System RelBib. Danke!

**Comment by KStelzel** [comment]
@coeh Ich stimme auch zu.

**Comment by coeh** [comment]
Vielen Dank! Da (wie auf Anregung von @thefass per Mail geklärt) @IxTheoKm auch bibliothekarisch kein Problem sieht, kann das nun umgesetzt werden. Kannst du @mtrojan-ub dich darum kümmern? Es ist nichts Technisches zu machen, es geht tatsächlich nur um den Wortlaut der betreffenden Unterüberschrift in allen drei Systemen.

**Comment by mtrojan-ub** [comment]
Zu diesem Text gibt es insgesamt 3 verschiedene Sprachvarianten.

DE: "Verbundene Autoren"
EN: "Related Authors"
FR: "Auteurs liés"

Sollen die EN/FR Übersetzungen ebenfalls angepasst werden?
